docs/README.md

```diff
@@ -115,7 +115,10 @@ overview of the compiler goals and architecture.
 ## Documentation Comments Style Guide 
 - Use triple slashes `///` for documenting functions and classes in files.
 - Double slashes `//` should be used for "internal" comments within functions.
+- Double slashes `//` should be used for inline comment.
 - For rare occasions such as adding comments to multi-line macros, you may use `/* ... */` style comments.
+- There should be no space at the end of the comment.
+- First latter of comment should be Capital letter.
```

Comment:
```suggestion
- First letter of the comment should be a capital letter.
```

Reply:
Also add that each comment should end with a fullstop/period. Please also do this for the comments in this PR.

Reply:
Any other suggestion that we can implement?